dev-python/pipenv/pipenv-2020.5.28.ebuild

```diff
@@ -5,6 +5,7 @@ EAPI=7
 
 PYTHON_COMPAT=( python3_{6,7,8} )
 
+DISTUTILS_USE_SETUPTOOLS="bdepend"
```

Comment:
That's the default.

Reply:
Yes, but I thought, it's fine to have this explicit. Do you want me to remove this?